plugin/src/test/kotlin/com/github/weisj/darkmode/platform/linux/gnome/GnomeNativeTest.kt

```diff
@@ -43,9 +43,10 @@ class GnomeNativeTest {
 
     private fun currentGtkTheme(): String {
         val theme = "gsettings get $settingsPath $settingsKey".runCommand()
+        //val theme = "xfconf-query -c xsettings -p /Net/ThemeName".runCommand()
```

Comment:
Why is this commented out?

Reply:
on my computer I use `xfconf-query` from XFCE but I don't see the need to change the default test based on gnome.

I left it as a comment to show that alternative utilities would also work.

Reply:
As these comments are removed in the folowing commit please drop them from this one :)